docs/content/guides/operations/kubernetes/kubernetes-metadata/index.md

```diff
@@ -42,6 +42,7 @@ Kubernetes metadata can be applied at the environment, application, or container
 
 - Applications.Core/containers
 - Applications.Core/gateways
+- Applications.Core/httpRoutes
```

Comment:
```suggestion
```

Even though there are some codepaths with httpRoutes we're not documenting them outside of autogenerated docs

Reply:
Removed